use-location.js

```diff
@@ -26,6 +26,14 @@ const subscribeToLocationUpdates = (callback) => {
   };
 };
 
+const getHistoryState = () => {
+  try {
+    return history.state;
+  } catch (error) {
```

Comment:
Is wrapping this line in `try..catch` really needed? What are the cases when it could throw?

Reply:
without **`try..catch`** it works totally fine on browser but throws error when you run the test cases `npm run test`